src/main/scala/tile/Core.scala

```diff
@@ -54,6 +57,7 @@ trait CoreParams {
   def customCSRs(implicit p: Parameters): CustomCSRs = new CustomCSRs
 
   def hasSupervisorMode: Boolean = useSupervisor || useVM
+  def hasBitManipCrypto: Boolean = useBitManipCrypto || useCryptoNIST || useCryptoSM
   def instBytes: Int = instBits / 8
```

Comment:
Can we add checks for invalid configurations? Is `useBitManipCrypto && !useBitManip` legal?

Reply:
Yes it is legal. People may do not want zba/zbs extension introduced by useBitManip hence only enabling useBitManipCrypto.